src/DDoSX/HstsClient.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace UKFast\SDK\DDoSX;
+
+use UKFast\SDK\DDoSX\Entities\Hsts;
+use UKFast\SDK\DDoSX\Entities\HstsRule;
+
+class HstsClient extends Client
+{
+    const HSTS_MAP = [];
+    const HSTS_RULE_MAP = [
+        'max_age' => 'maxAge',
+        'include_subdomains' => 'includeSubdomains',
+        'record_name' => 'recordName'
+    ];
+
```

Comment:
Adding a docblock to this public method (as you have done with the other public methods), would allow the IDE to be more helpful for the developer